app/controllers/api/users.php

```diff
@@ -1033,7 +1033,7 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_USER)
     ->param('userId', '', new UID(), 'User ID.')
-    ->param('name', '', new Text(128), 'User name. Max length: 128 chars.')
+    ->param('name', '', new Text(128, 0), 'User name. Max length: 128 chars.')
```

Comment:
Yea this PR should have only made this possible.. to allow empty names. The other modifications don't feel secure and opens up the attack surface for more vulnerabilities